runtime/onert/backend/cpu/ops/SplitLayer.h

```diff
@@ -36,9 +36,10 @@ class SplitLayer : public ::onert::exec::IFunction
   SplitLayer();
 
 public:
-  void splitFloat32();
+  // void splitFloat32();
 
-  void splitQuant8();
+  // void splitQuant8();
```

Comment:
@dr-venkman Please remove this lines instead of commenting out. Except for this, LGTM.

Reply:
Thanks for the comment. I will update the code with the suggested changes.